runelite-client/src/main/java/net/runelite/client/plugins/chathistory/ChatHistoryPlugin.java

```diff
@@ -42,12 +43,18 @@
 {
 	private static final String WELCOME_MESSAGE = "Welcome to RuneScape.";
 	private static final Set<ChatMessageType> ALLOWED_HISTORY = Sets.newHashSet(
-		ChatMessageType.PUBLIC,
-		ChatMessageType.CLANCHAT,
-		ChatMessageType.PRIVATE_MESSAGE_RECEIVED,
-		ChatMessageType.PRIVATE_MESSAGE_SENT,
-		ChatMessageType.PRIVATE_MESSAGE_RECEIVED_MOD,
-		ChatMessageType.GAME
+			ChatMessageType.PUBLIC,
```

Comment:
I believe the indent before was correct, but depends on IDE i guess. Still, should be fixed